### ISSUE TITLE
Consider removing awt dependency

### ISSUE THREAD

**Issue by SilverHatTrick** [TeamLapen/Vampirism#956]
https://pastebin.com/4d7PHTz8

**Comment by yavincl** [comment]
```
yago@vault ~ $ find /usr/lib/ | grep -i libxrender
/usr/lib/libXrender.so
/usr/lib/libXrender.so.1
/usr/lib/libXrender.so.1.3.0
yago@vault ~ $ pacman -F /usr/lib/libXrender.so.1 
usr/lib/libXrender.so.1 is owned by extra/libxrender 0.9.10-4
```

Install libxrender or your distribution's equivalent. To see if libxrender is installed do `find /usr/lib/ | grep -i libxrender`

If yes you will get output

**Comment by SilverHatTrick** [comment]
In the console?

**Comment by maxanier** [comment]
Interesting, I did not know that Java awt wasn't always available/supported.

I assume you are running on a headless server. Do you have root access to it or is it a hosted MC server by some server hoster?
If its the first you can follow yagoplx suggestions in the server linux console, if its the latter please let me know which hoster you are using

**Comment by SilverHatTrick** [comment]
> 
> 
> Interesting, I did not know that Java awt wasn't always available.
> 
> I assume you are running on a headless server. Do you have root access to it or is it a hosted MC server by some server hoster?
> If its the first you can follow yagoplx suggestions in the server linux console, if its the latter please let me know which hoster you are using

I am using AliHosting!

https://www.alishosting.com/

**Comment by maxanier** [comment]
Mhh, you can try to contact them and ask them if they can install Java awt / libxrender or if they know a different workaround.
Unfortunately, I cannot get rid of the specific part of code as it is part of the API. I can only do this with 1.17.
Also, it has been used in Vampirism for years and this is the first time such an issue occured

**Comment by Cheaterpaul** [comment]
was fixed in 97ed4ff1109c862da482ebc6db63e0b491ffba33
